zynqmp/optee-client/optee-client.bb

```diff
@@ -0,0 +1,57 @@
+OPTEE_VERSION ??= "latest"
+SRCREV ??= "${AUTOREV}"
+BRANCH ??= "master"
+
+DESCRIPTION = "OP-TEE Client"
+
+#defined as closed license to prevent MD5 checksum verification
+LICENSE = "CLOSED"
+
+PROVIDES = "virtual/optee-client"
+
+inherit deploy
+
+DEPENDS += "virtual/optee-os"
+
+S = "${WORKDIR}/git"
+B = "${WORKDIR}/build"
+
+PV = "${OPTEE_VERSION}+git${SRCPV}"
+
+REPO ??= "git://github.com/OP-TEE/optee_client.git;protocol=https"
+BRANCHARG = "${@['nobranch=1', 'branch=${BRANCH}'][d.getVar('BRANCH', True) != '']}"
+SRC_URI = "${REPO};${BRANCHARG} \
+	"
+
+OPTEE_BASE_NAME ?= "${PN}-${PKGE}-${PKGV}-${PKGR}-${DATETIME}"
+OPTEE_BASE_NAME[vardepsexclude] = "DATETIME"
+
+COMPATIBLE_MACHINE = "zynqmp"
+PLATFORM_zynqmp = "zynqmp"
+
+# requires CROSS_COMPILE set by hand as there is no configure script
+export CROSS_COMPILE="${TARGET_PREFIX}"
+
+EXPORT_DIR = "${TMPDIR}/deploy/images/${MACHINE}/optee/export_client"
+
+EXTRA_OEMAKE_append = " CROSS_COMPILE=${CROSS_COMPILE}"
+EXTRA_OEMAKE_append = " DESTDIR=${EXPORT_DIR}"
+EXTRA_OEMAKE_append = " SBINDIR=/sbin"
+EXTRA_OEMAKE_append = " LIBDIR=/lib"
+EXTRA_OEMAKE_append = " INCLUDEDIR=/include"
+
+do_configure() {
+	oe_runmake clean -C ${S} clean
+}
+
+do_compile() {
+	oe_runmake -C ${S} build
```

Comment:
minor: This looks like the default oe make way. Do you think you can remove this local `do_compile` definition (maybe the configure step also)?

Reply:
Actually i have no idea. My guess is no and i can't find anything about default do_compile or do_configure target in Yocto documentation. It is only said that this target are automatically called when a bitbake recipe is executed. If someone have more informartion can change it otherwise i will leave it as it is.